tests/plugins/lti_consumer/test_models.py

```diff
@@ -32,6 +32,43 @@ def test_lti_consumer_models_is_automatic_resizing_default(self):
         self.assertTrue(instance.is_automatic_resizing)
         self.assertEqual(instance.inline_ratio, 0.5625)
 
+    def test_lti_consumer_models_default_with_lti_provider(self):
+        """
+        The "inline_ratio" and "is_automatic_resizing" fields should default to the
+        lti provider values if there are otherwise respectively to 0.5625 and True.
```

Comment:
I don't see how is this superior to getting it from the settings? We are going to replicate the info in database and won't be able to change it for all at one?